test/db/feeder_transistor.sql

```diff
@@ -0,0 +1,941 @@
+--
+
+CREATE SEQUENCE episodes_id_seq
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE episodes_id_seq OWNER TO feeder;
+
+--
+-- Name: episodes_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: feeder
+--
+
+ALTER SEQUENCE episodes_id_seq OWNED BY episodes.id;
+
+
+--
+-- Name: feed_images; Type: TABLE; Schema: public; Owner: feeder
+--
+
+CREATE TABLE feed_images (
+    id integer NOT NULL,
+    url character varying,
+    link character varying,
+    description character varying,
+    height integer,
+    width integer,
+    podcast_id integer,
+    title character varying,
+    format character varying,
+    size integer
+);
+
+
+ALTER TABLE feed_images OWNER TO feeder;
+
+--
+-- Name: feed_images_id_seq; Type: SEQUENCE; Schema: public; Owner: feeder
+--
+
+CREATE SEQUENCE feed_images_id_seq
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE feed_images_id_seq OWNER TO feeder;
+
+--
+-- Name: feed_images_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: feeder
+--
+
+ALTER SEQUENCE feed_images_id_seq OWNED BY feed_images.id;
+
+
+--
+-- Name: itunes_categories; Type: TABLE; Schema: public; Owner: feeder
+--
+
+CREATE TABLE itunes_categories (
+    id integer NOT NULL,
+    created_at timestamp without time zone,
+    updated_at timestamp without time zone,
+    podcast_id integer,
+    name character varying NOT NULL,
+    subcategories character varying
+);
+
+
+ALTER TABLE itunes_categories OWNER TO feeder;
+
+--
+-- Name: itunes_categories_id_seq; Type: SEQUENCE; Schema: public; Owner: feeder
+--
+
+CREATE SEQUENCE itunes_categories_id_seq
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE itunes_categories_id_seq OWNER TO feeder;
+
+--
+-- Name: itunes_categories_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: feeder
+--
+
+ALTER SEQUENCE itunes_categories_id_seq OWNED BY itunes_categories.id;
+
+
+--
+-- Name: itunes_images; Type: TABLE; Schema: public; Owner: feeder
+--
+
+CREATE TABLE itunes_images (
+    id integer NOT NULL,
+    url character varying,
+    podcast_id integer,
+    format character varying,
+    width integer,
+    height integer,
+    size integer
+);
+
+
+ALTER TABLE itunes_images OWNER TO feeder;
+
+--
+-- Name: itunes_images_id_seq; Type: SEQUENCE; Schema: public; Owner: feeder
+--
+
+CREATE SEQUENCE itunes_images_id_seq
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE itunes_images_id_seq OWNER TO feeder;
+
+--
+-- Name: itunes_images_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: feeder
+--
+
+ALTER SEQUENCE itunes_images_id_seq OWNED BY itunes_images.id;
+
+
+--
+-- Name: media_resources; Type: TABLE; Schema: public; Owner: feeder
+--
+
+CREATE TABLE media_resources (
+    id integer NOT NULL,
+    episode_id integer,
+    "position" integer,
```

Comment:
is "position" a reserved word? (why quotes?)

Reply:
idk - this was generated by the `pg_dump` utility, I did not write this.